src/transformers/utils/versions.py

```diff
@@ -22,7 +22,12 @@
 
 from packaging import version
 
-import pkg_resources
+
+# The package importlib_metadata is in a different place, depending on the python version.
+if sys.version_info < (3, 8):
+    import importlib_metadata
+else:
+    import importlib.metadata as importlib_metadata
```

Comment:
We probably should do this if/else only once. And this is already done in `file_utils.py`.

May be let's make `src/transformers/utils/versions.py` the importer as you did here and then import `from transformers.utils.versions import importlib_metadata` in `file_utils.py` and in the test below and anywhere else it might be needed?

Reply:
Done